ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/model/response/BaseResponse.java

```diff
@@ -21,15 +21,13 @@
 
 public class BaseResponse {
 
-    private final NetHelper netHelper;
+    private final NetHelper netHelper = NetHelper.getInstance();
```

Comment:
I didn't see the need for passing in the net helper. I can change it back if you prefer